### PR TITLE
Issue 302

### LINKED ISSUE #302: Errors while downloading sources
Hi,
I would like to ask if there is a way how to make automatic scratch builds
work if you have sources that are not located at some url but are directly in the
dist git. For example here https://bugzilla.redhat.com/show_bug.cgi?id=1835164
scratch build was not executed because the-now-hotness could not
download memcached.sysconfig which is a file stored directly in dist git and thus
does not need to be downloaded.

### PR DESCRIPTION
**Pull request by TomasKorbar**
Hi @Zlopez,
I decided to try to help you help me:)
So i fixed issue 302 with the commit https://github.com/fedora-infra/the-new-hotness/commit/03b2f6f9b48402fb1ae1b428a71a0398753391bd .
For more information please see commit messages.
Feel free to propose any change.

Closes #302

### CODE REVIEW

**Comment by Zlopez** [comment]
I noticed that your PR is missing news file, could you add one.
You can look at how to add it in [docs](https://the-new-hotness.readthedocs.io/en/latest/dev-guide.html#release-notes).

**Comment by TomasKorbar** [comment]
> I noticed that your PR is missing news file, could you add one.
> You can look at how to add it in [docs](https://the-new-hotness.readthedocs.io/en/latest/dev-guide.html#release-notes).

Fixed

File: news/302.bug [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Errors while downloading sources
```
Comment (Zlopez) [inline news/302.bug:1]: It is better if the news file has only one line. This will be used to generate changelog.

Comment (TomasKorbar) [reply news/302.bug:1]: Fixed.

**Pull request merged**